src/utils/typography.js

```diff
@@ -1,6 +1,8 @@
 import Typography from "typography"
 
 const typography = new Typography({
+  baseFontSize: "14px",
```

Comment:
Let's try to use `var(--layout-base)` and `calc()` here?

Reply:
The file shows sintaxe erros.